ui/ducks/bridge/actions.ts

```diff
@@ -70,3 +104,356 @@ export const setToChain = (chainId: Hex) => {
     );
   };
 };
+
+export const getBridgeERC20Allowance = async (
+  contractAddress: string,
+  walletAddress: string,
+  chainId: Hex,
+): ReturnType<BridgeController['getBridgeERC20Allowance']> => {
+  return await submitRequestToBackground(
+    BridgeBackgroundAction.GET_BRIDGE_ERC20_ALLOWANCE,
+    [contractAddress, walletAddress, chainId],
+  );
+};
+
+export const submitBridgeTransaction = (
+  history: ReturnType<typeof useHistory>,
+) => {
+  return async (
+    dispatch: MetaMaskReduxDispatch,
+    getState: () => MetaMaskReduxState & BridgeAppState,
+  ) => {
+    const state = getState();
+    const isBridgeEnabled = getIsBridgeEnabled(state);
+    const isBridgeChain = getIsBridgeChain(state);
+
+    if (!(isBridgeEnabled && isBridgeChain)) {
+      // TODO do we want to do something here?
+      return;
+    }
```

Comment:
We check these before routing to the bridge page so maybe not necessary to return early?